dropwizard-auth/src/main/java/io/dropwizard/auth/CachingAuthorizer.java

```diff
@@ -41,7 +41,9 @@
     //
     // `null` cache values are interpreted as cache misses, and will
     // thus result in read through to the underlying `Authorizer`.
-    private final LoadingCache<ImmutablePair<P, String>, Boolean> cache;
+    //
+    // Field is package-private to be visible for unit tests
+    final LoadingCache<ImmutablePair<P, String>, Boolean> cache;
```

Comment:
Please add [`@VisibleForTesting`](https://google.github.io/guava/releases/27.0.1-jre/api/docs/com/google/common/annotations/VisibleForTesting.html) to the field.

Reply:
Yeah I wasn't sure about adding that since it looks like you're removing Guava. Should I use the one in `com.google.common.annotations` or `io.dropwizard.logback.shaded.guava.annotations`?

Reply:
Use the one from Guava for now, please. 😃